packages/create-guten-block/app/npmInstallScripts.js

```diff
@@ -45,7 +45,7 @@ module.exports = ( blockName, blockDir, isCanary ) => {
 	}
 	// Install latest exact version of cgb-scripts.
 	return new Promise( async resolve => {
-		await execa( 'npm', [ 'install', 'cgb-scripts', '--save', '--save-exact', '--silent' ] );
+		await execa( 'npm', [ 'install', '../packages/cgb-scripts', '--save', '--save-exact', '--silent' ] );
```

Comment:
This doesn't look right. I have added you as a maintainer of npm packages. Publish `canary` packages and test with that using the Contributing.md guide in this repo. Don't publish a public release without approval here.